src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -36,6 +37,8 @@ public class JwtSecurityTokenHandler : SecurityTokenHandler
         private static string _shortClaimType = _namespace + "/ShortTypeName";
         private bool _mapInboundClaims = DefaultMapInboundClaims;
 
+        internal ITelemetryInstrumentation _telemetryClient = new TelemetryInstrumentation();
```

Comment:
There is a naming oddity here.
Should ITelemetryInstrumentation be ITelemeteryClient?
Then TelemetryInstrumentation be TelemetryClient?